syne_tune/blackbox_repository/conversion_scripts/scripts/fcnet_import.py

```diff
@@ -197,6 +202,8 @@ def generate_fcnet():
                 resource_attr: RESOURCE_ATTR,
             },
         )
+    with catchtime("compute hash"):
+        store_hash(__file__, repository_path / blackbox_name)
```

Comment:
Which hashes would we compare then? If the code changes, the generated files stay the same as long as a users does not explicitly rebuild the code.

Reply:
I agree, we should hash files, not code. The hash to compare against will be part of the code. You update your code to generate different files, you will also have to update your hash.

Why not regenerate files directly instead of giving a warning?

Reply:
> Which hashes would we compare then? 

We can compare with all the hashes of the different files generated (objectives, config-space, hyperparameters etc). 

I also agree with Martin that regenerating may be safer than just giving a warning.

Reply:
I would probably compute all those hashes put add them to a set and compute its hash so we only need to handle a single hash value in our code. for some benchmarks (looking at pd1), the amount of files is quite big.